data/low_income_thresholds.json

```diff
@@ -1039,6 +1039,64 @@
       }
     }
   },
+  "ME": {
+    "me-low-income": {
+      "type": "hhsize",
+      "incentives": [
+        "ME-1",
+        "ME-7",
+        "ME-9",
+        "ME-10",
+        "ME-14",
+        "ME-17",
+        "ME-18"
+      ],
+      "/": "recipients of HEAP, SNAP, TANF, and/or MaineCare qualify; HEAP has the highest income limits of these programs",
+      "source_url": "https://www.mainehousing.org/charts/HEAP-Income-Elibility",
+      "thresholds": {
+        "1": 35336,
+        "2": 46209,
+        "3": 57082,
+        "4": 67955,
+        "5": 78827,
+        "6": 89700,
+        "7": 91739,
+        "8": 93777
+      }
+    },
+    "me-moderate-income": {
+      "type": "filing-status",
+      "incentives": [
+        "ME-2",
+        "ME-6",
+        "ME-8",
+        "ME-15"
+      ],
+      "source_url": "https://www.efficiencymaine.com/docs/Rebates-Income-Eligible-Households.pdf",
+      "thresholds": {
+        "hoh": [
+          0,
+          70000
+        ],
+        "joint": [
+          0,
+          100000
+        ],
+        "single": [
+          0,
+          70000
+        ],
+        "married_filing_separately": [
+          0,
+          70000
+        ],
+        "qualifying_widower_with_dependent_child": [
+          0,
+          70000
+        ]
```

Comment:
I know the source PDF just says the limit for "joint filers" is 100k, but I think that would apply to this status as well. It's meant for people who used to file jointly with their spouse until their spouse died. (Also known as Qualifying Surviving Spouse or QSS.)